### PR TITLE
Update some broken links in the FAQ docs

### PR DESCRIPTION
**Pull request by cmintey**
## What type of PR is this?

- documentation

## What this PR does / why we need it:

Cleans up some broken links in the FAQs and removes the section on language because I don't think that relevant anymore?

### CODE REVIEW

File: docs/docs/overrides/api.html
Comment (boc-the-git) [inline docs/docs/overrides/api.html:1]: What's the impact/intent of this change?

Comment (Kuchenpirat) [reply docs/docs/overrides/api.html:1]: Its get generated when running `make docs` and fills the API Reference

**Review by Kuchenpirat — APPROVED** [review]
LGTM.

**Review by boc-the-git — APPROVED** [review]
I love a good fix/improvement to the docs 👍

**Pull request merged**